src/lib/remark-plugins/rewrite-tutorial-links/utils/handle-learn-links.ts

```diff
@@ -48,8 +50,15 @@ export function handleTutorialLink(
 	let path = ''
 	const collectionSlugParam = url.searchParams.get('in')
 	if (collectionSlugParam) {
-		const collectionSlug = collectionSlugParam.split('/')[1]
-		path = `/${product}/tutorials/${collectionSlug}/${filename}`
+		// the collection product should take priority if the `in` query is used
+		const [collectionProduct, collectionFilename] =
```

Comment:
Fix here: It's important that the collection product retains the top level context when the 'in' param is passed.